src/python/zquantum/core/wip/circuits/conversions/cirq_conversions.py

```diff
@@ -139,19 +139,22 @@ def _rotation(angle: Parameter) -> cirq.EigenGate:
 qubit_index = attrgetter("x")
 
 
-@overload
-def export_to_cirq(gate: _gates.Gate) -> cirq.Gate:
-    pass
+# The overloads are commented out because mypy does not yet support overloads
```

Comment:
I'm not sure if that's a good enough reason to remove these functions 🤔

Reply:
Overloads are only for type checking, they don't actually affect the behavior of the function. Though maybe there's a more elegant way though to tell mypy to ignore them besides commenting them out.

Reply:
I'm not sure if we don't have a clash between [this overload](https://www.python.org/dev/peps/pep-3124/) and [this overload](https://pypi.org/project/overload/) here.
Cause I'm pretty confident that I have used the latter.

Reply:
Ohh that is confusing. The cirq conversions are importing `overload` from `typing` and not the `overloading` module... @dexter2206 can you confirm that this was what was intended?

Reply:
Added a facade to overcome the mypy errors (01b236a). Mypy still reports some errors in `pyquil_conversions`, I'll look into it later